crates/modelardb_manager/src/main.rs

```diff
@@ -42,13 +42,8 @@ use crate::metadata::MetadataManager;
 use crate::remote::start_apache_arrow_flight_server;
 
 /// The port of the Apache Arrow Flight Server. If the environment variable is not set, 8888 is used.
-pub static PORT: Lazy<u16> = Lazy::new(|| match env::var("MODELARDBM_PORT") {
-    Ok(port) => port
-        .parse()
-        .map_err(|_| "MODELARDBM_PORT must be between 1 and 65535.")
-        .unwrap(),
-    Err(_) => 8888,
-});
+pub static PORT: Lazy<u16> =
+    Lazy::new(|| env::var("MODELARDBM_PORT").map_or(8888, |value| value.parse().unwrap()));
```

Comment:
Maybe use a port adjacent to `modelardbd`s default port to make it simpler to remember the ports used by the different components of ModelarDB. Also the commonly used Jupyter notebooks [use port 8888 by default](https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers) so maybe use a different pot to allow both to be run together.

Reply:
Fixed.